src/pages/Home/index.tsx

```diff
@@ -56,30 +56,34 @@ function Home() {
     });
   }, []);
 
-  const handleCheckedChange = (checkedChangedValues: string[]) => {
-    setCheckedValues(checkedChangedValues);
-  };
+  useEffect(() => {
+    const handleResize = () => {
+      navbar?.setTitle(window.innerWidth >= 900 ? 'Top CS Conference Deadlines' : 'Conference Deadlines');
+    };
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, [navbar]);
```

Comment:
There is the same code in the ConferencePage to set the logo.
Both things should be in the same place